UI/src/app/feedback-button/feedback-button.component.html

```diff
@@ -0,0 +1,8 @@
+<button 
+    aria-label="Give feedback about LitPaths" 
+    class="px-2 py-1 border-gray-600 focus-within:border-yellow-600 border-solid border-2 rounded-md"
+    [class.bg-gray-100]=true
+    [class.text-white]=false
+    (click)="giveFeedback()">
```

Comment:
Do you need to have a function to do this?

In general, try to avoid writing code that the browser already does such as links, so you could make this an `<a>` tag with a href to the form.

Reply:
Oops you're right!

Reply:
Yes true, I was just thinking about keeping it consisent for future use, but we don't need it right now